cmd/list.go

```diff
@@ -52,6 +60,8 @@ func doList(cmd *cobra.Command, args []string) error {
 
 	var v string
 
+	secretList := []Secret{}
```

Comment:
It seems most of Go code use the plural form to indicate slice, like `secrets`.

How about renaming this to `secrets`, and rename _existing_ `secrets` (the result of `k8sclient.ListSecrets()`) to `ss` ?
IMO it's easy-to understand the variable scope.

Reply:
Fixed.